vertical-pod-autoscaler/pkg/recommender/input/cluster_feeder.go

```diff
@@ -175,11 +175,11 @@ func newPodClients(kubeClient kube_client.Interface, resourceEventHandler cache.
 	// We are interested in pods which are Running or Unknown (in case the pod is
 	// running but there are some transient errors we don't want to delete it from
 	// our model).
-	// We don't want to watch Pending, Succeeded, or Failed failed pods because we
-	// know they're not generating any usage.
-	// Filter
-	statusFilter := fmt.Sprintf("status.phase!=%s,status.phase!=%s,status.phase!=%s", apiv1.PodPending, apiv1.PodSucceeded, apiv1.PodFailed)
-	selector := fields.ParseSelectorOrDie(statusFilter)
+	// We don't want to watch Pending pods because they didn't generate any usage
+	// yet.
+	// Succeeded and Failed failed pods don't generate any usage anymore but they
```

Comment:
Can you rephrase the comment to say what data we lose when we stopped listing those pods? 
It's also a bit worrying since I'm not sure if API server gives any guarantees on how long the failed and succeeded pods are kept for.

Reply:
Rephrased. I didn't see any deleted data (that we wouldn't want to delete). I think we won't delete any. I just want to highlight here that one needs to be careful if they decide to change this